stdlib/experimental/influxdb/influxdb_test.go

```diff
@@ -0,0 +1,238 @@
+package influxdb
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func Test_api(t *testing.T) {
+	for _, test := range []struct {
+		name string
+
+		// passed to api func
+		args values.Object
+
+		// expected to be returned from api func call
+		expectedErrorMessage string
+		expectedStatusCode   int64
+		expectedResponseBody []byte
+
+		// status and responses to be returned from mock api host
+		status   int
+		response []byte
+
+		// method, status, token and paths mock api host expects to receive
+		expectedMethod         string
+		expectedRequestBody    []byte
+		expectedRequestPath    string
+		expectedRequestToken   string
+		expectedRequestHeaders http.Header
```

Comment:
These look great, can you also add tests for the expected response headers? Since the `api` type signature returns the headers we should assert that code path is working.